ledger-test-tool/suites/src/main/scala/com/daml/ledger/api/testtool/suites/v1_15/InterfaceSubscriptionsIT.scala

```diff
@@ -808,7 +914,10 @@ class InterfaceSubscriptionsIT extends LedgerTestSuite {
       createdEventWithBlob.templateId.get.toString,
       Tag.unwrap(T1.id).toString,
     )
-    assertEquals(createdEventWithBlob.createArgumentsBlob.nonEmpty, createArgumentsBlobNonEmpty)
+    val blobPresence =
+      if (useTemplateIdBasedLegacyFormat) createdEventWithBlob.createArgumentsBlob.nonEmpty
+      else !createdEventWithBlob.createdEventBlob.isEmpty
+    assertEquals(blobPresence, createArgumentsBlobNonEmpty)
```

Comment:
A bit confusing tbh. Maybe rename the argument name of the method to `createBlobNonEmpty` instead of `createArgumentsBlobNonEmpty` since that's specific

Reply:
renamed to `expectBlobPresent`